### PR TITLE
v2, podman: fix and enable all run_userns_test.go tests

### PR DESCRIPTION
**Pull request by giuseppe**
more details in each commit.

Signed-off-by: Giuseppe Scrivano <gscrivan@redhat.com>

### CODE REVIEW

File: pkg/namespaces/namespaces.go [hunk 1]
```diff
@@ -102,6 +102,11 @@ func (n UsernsMode) IsAuto() bool {
 	return parts[0] == "auto"
 }
 
+// IsDefaultValue indicates whether the user namespace has the default value.
+func (n UsernsMode) IsDefaultValue() bool {
```
Comment (mheon) [inline pkg/namespaces/namespaces.go:106]: Shouldn't this also check for the explicit Default constant?

Comment (giuseppe) [reply pkg/namespaces/namespaces.go:106]: I've copied it from cgroupns.

I've fixed both and pushed a new version

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

**Pull request merged**